<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SerialTerminal</class>
 <widget class="QMainWindow" name="SerialTerminal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>477</width>
    <height>321</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>477</width>
    <height>321</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>477</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Network Configuration</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QComboBox" name="COM_comboBox">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Connect_pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>231</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Refresh_pushButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>10</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="MAC_lineEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>80</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="MAC_label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>80</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>MAC :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="GTW_lineEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>110</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="GTW_label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Gateway :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="SMSK_lineEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>140</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="SMSK_label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Sub MSK :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="IP_lineEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>170</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="IP_label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>IP Adress:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="PORT_lineEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>200</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>5</number>
    </property>
   </widget>
   <widget class="QLabel" name="PORT_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Server Port :</string>
    </property>
   </widget>
   <widget class="QPushButton" name="RestoreDefault_pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>201</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Default settings</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Write_pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>201</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Assign network settings</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>477</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
